include/Contact.php

```diff
@@ -662,4 +664,34 @@ function posts_from_contact($a, $contact_id) {
 
 	return $o;
 }
+
+/**
+ * @brief Returns a formatted location string from the given profile array
+ *
+ * @param array $profile Profile array (Generated from the "profile" table)
+ *
+ * @return string Location string
+ */
+function formatted_location($profile) {
```

Comment:
I'm not sure about that. We really have to move many functions to better places. I would like to split the large files into many small files as well. I thought the "Contact.php" would be the right place because the function is used for contact data.